lighthouse-cli/test/static-server-test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const http = require('http');
+const {server} = require('./fixtures/static-server.js');
+
+/**
+ * @param {string} url
+ */
+function get(url) {
```

Comment:
fwiw, we already have `isomorphic-fetch` in our devDeps that should make this much easier :)

```js
const fetch = require('isomorphic-fetch')
const get = url => fetch(url).then(r => r.text())
```

Reply:
so fetch